docs/help-topics/rhtap-secrets/rhtap-secrets.yml

```diff
@@ -0,0 +1,44 @@
+# Name is an internal name. Title displays in the UI as a side panel title.
+# Leave tags empty for now. Tags will specify where in the app descriptions will be available.
+# Links to be external only. We don't know yet whether referencing other side panels will be supported but referencing to in-depth docs is expected to be supported.
+# Note that this topic is for LEARN MORE.
+
+- name: rhtap-secrets-secrets
+  tags:
+  title: Secrets
+  content: |-
+    Add and manage your build and deployment secrets and their related configurations. You can add a secret at the workspace level to keep your data private. All secrets are stored using AWS Secrets Manager.  
+
+    ## Secrets storage in AWS Secrets Manager
+    All of your secrets are stored in AWS Secrets Manager to make sure that only applications within your workspace can access them. For privacy, avoid using environment variables to store your secrets because they send the contents of the secret back to publicly accessible repositories. 
+
+    ## Deployment secrets
+    Deployment secrets are useful after you’ve built and deployed your application and it’s running in an environment. Specifically, if your application relies on a database or other service, you can use a deployment secret to require and provide credentials to connect to these various services. With a deployment secret, you can have both a staging database and a production database and be able to control access to each one individually.
+
+    ## Source secrets for private repositories and parent images
+    You can configure a source secret to give a builder pod access to your private repositories or parent images. With access to internal systems and services, the source secret fetches private repositories and parent images, providing a more secure framework.  
+
+    ## Build secrets
+    Build secrets facilitate authentication because they provide protected data, like credentials, to tasks in a component’s build pipeline.
+
+      - Using Snyk
+        A secret can provide Snyk with an authentication code so that Snyk can test each component build for vulnerabilities. If your component builds from an image in a private registry, you can also use a secret to provide Snyk with an access token so that it can pull that image. **Note:** RHTAP includes a Snyk task for static application security testing (SAST) scanning, but you must provide your own token to enable it. If RHTAP can’t find a token, it skips the Snyk task.
```

Comment:
Ignore me if this isn't the case, but do we have to use {ProductName} in microcopy? Also, I thought we weren't supposed to abbreviate RHTAP ever